Agro_easy/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import About from './components/about';
+import Home from './components/home';
+
+
```

Comment:
Add a comment to describe this component. Use this as a reference (https://react-styleguidist.js.org/docs/documenting.html)